packages/core/bootstrap/src/lib/config/provider-limits/config.ts

```diff
@@ -30,8 +36,33 @@ export function get(
   const perMinuteRateLimit = getEnv('RATE_LIMIT_CAPACITY_MINUTE')
   const shouldIgnorePerSecLimit = perSecRateLimit && parseInt(perSecRateLimit) <= 0
   const shouldIgnorePerMinLimit = perMinuteRateLimit && parseInt(perMinuteRateLimit) <= 0
-  if (perSecRateLimit) capacity = shouldIgnorePerSecLimit ? 0 : parseInt(perSecRateLimit)
-  if (perMinuteRateLimit) capacity = shouldIgnorePerMinLimit ? 0 : parseInt(perMinuteRateLimit)
+  let highestTierLimit = 0
+
+  if (perSecRateLimit) {
+    capacity = shouldIgnorePerSecLimit ? 0 : parseInt(perSecRateLimit)
+    highestTierLimit = getLastTierLimitValue(rateLimitConfig.limits, 'http', 'rateLimit1s')
+  }
+
+  if (perMinuteRateLimit) {
+    capacity = shouldIgnorePerMinLimit ? 0 : parseInt(perMinuteRateLimit)
+    highestTierLimit = getLastTierLimitValue(rateLimitConfig.limits, 'http', 'rateLimit1m')
+  }
+
+  if (enabled && capacity > highestTierLimit) {
+    // logger.warn(
+    //   `The configured RATE_LIMIT_CAPACITY value is higher than the highest tier value from limits.json`,
+    // )
+    logger.warn(
+      `The configured ${
+        perSecRateLimit
+          ? 'RATE_LIMIT_CAPACITY_SECOND'
+          : perMinuteRateLimit
+          ? 'RATE_LIMIT_CAPACITY_MINUTE'
+          : 'RATE_LIMIT_CAPACITY'
```

Comment:
Sorry, one last thing. Could you flip the order of second and minute in this conditional. Based on the logic above, the minute limit would overwrite the capacity. So if both limits are set the minute capacity would trigger this warning but the message would say RATE_LIMIT_CAPACITY_SECOND.
```suggestion
          perMinuteRateLimit
          ? 'RATE_LIMIT_CAPACITY_MINUTE'
          : perSecRateLimit
          ? 'RATE_LIMIT_CAPACITY_SECOND'
          : 'RATE_LIMIT_CAPACITY'
```

Reply:
done!